### PR TITLE
Nick grid

### PR DESCRIPTION
**Pull request by NicholasJV**
template loading stuff works!

### CODE REVIEW

File: browser/js/common/factories/TemplateFactory.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+app.factory("TemplateFactory", function ( $http, GridFactory ){
```
Comment (codebyuma) [inline browser/js/common/factories/TemplateFactory.js]: You're not using GridFactory, don't need to inject :)

File: browser/js/template-modal/template-modal-controller.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+app.controller('templateModalCtrl', function($scope, $uibModalInstance, TemplateFactory, allTemplates, GridFactory){
```
Comment (codebyuma) [inline browser/js/template-modal/template-modal-controller.js]: Can also remove GridFactory injection here

File: browser/js/common/factories/TemplateFactory.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+	var TemplateFactory = {};
+
+
+	TemplateFactory.getAll = function(){
+		console.log('=-=-=-=-= hitting get all templates in factory')
```
Comment (sarahedkins) [inline browser/js/common/factories/TemplateFactory.js]: Take out the console.log

File: seed.js [hunk 4]
```diff
@@ -55,4 +419,16 @@ connectToDb.then(function () {
         console.error(err);
         process.kill(1);
     });
+
+    clearDB();
```
Comment (codebyuma) [inline seed.js]: Do we still need this clear in our seed file?

**Pull request merged**